lib/inferno/utils/evaluator_util.rb

```diff
@@ -0,0 +1,137 @@
+module Inferno
+  module Utils
+    module EvaluatorUtil
+      # General util methods
+      def self.get_meta_profile(resource)
+        resource&.meta&.profile || []
+      end
+
+      # reference resolution methods
+      def self.extract_ids_references(resources)
+        extractor = ReferenceExtractor.new
+        extractor.extract_ids_references(resources)
+      end
+
+      def self.validate_resource(resource)
```

Comment:
This validate_resource function isn't used anywhere, and it's not that useful in its current form anyway, so this should be deleted. We'll integrate with the real Validator class when the time comes

Reply:
Removed